projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -346,6 +346,11 @@ public function add_appearance_menu() {
 	 * Adds Plugins menu.
 	 */
 	public function add_plugins_menu() {
+		$submenus_to_update = apply_filters( 'wpcom_plugins_submenu_update', array() );
```

Comment:
Could you add a docblock for this new filter? Here is an example:
https://github.com/Automattic/jetpack/blob/d4b7af2a663239fa37ffb564ac8d95715979749d/projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php#L260-L270